### DISCUSSION TITLE
资源画像和数据预测

### DISCUSSION THREAD

**Discussion by zwzhang0107** [koordinator-sh/koordinator#1941] [General]
/todo

**Comment by zxustc** [comment]
03.06 todo list
1. status feeder 代码流程 @zxustc
2. histogram checkpoint 机制：crd 定义、reconcile 等 @zxustc
3. 新版接口定义 @zwzhang0107

**Comment by zxustc** [comment]
03.20 todo list
新版接口定义 @zwzhang0107
status feeder 新版接口 ready 后 rebase @zxustc
histogram checkpoint 机制：crd 定义、接口定义 @xueche

**Comment by zxustc** [comment]
03.27 update
-- check list --
新版接口定义 @zwzhang0107  done
histogram crd 定义：@xueche done
-- todo list --
status feeder 新版接口 ready 后 rebase，pr，下次一起讨论 @zxustc
histogram checkpoint 机制：接口定义 @xueche done

**Comment by zxustc** [comment]
04.10 update
--- check list ---
1. 完成内部代码的接口流程定义；@zxustc
--- todo list ---
1. 框架代码提交到 社区发起一个 pr，check 思路是否一致；@zxustc
2. check point 接口定义和流程；@xueche
3. profiler 核心工作流程 @zwzhang0107

**Comment by zxustc** [comment]
04.18 update
--- check list [done] ---
1. 提交 frontend pr @zxustc 
2. checkpoint 接口第一次定义讨论 @xueche
3. profiler 核心工作流程开发 @zwzhang0107 
--- todo list ---
1. 优化 checkpoint 接口，提 pr；@xueche
4. frontend pr 修改以及 workload fetcher 提 pr；@zxustc
5. metric collector 的实现和pr； @zwzhang0107

**Comment by zxustc** [comment]
04.25


---check list---
1. 优化 checkpoint 接口，v2[done]@xueche
2. workload fetcher 自测中 @zxustc 
3. metric collector 开发中 @zwzhang0107 

---todo list---
1. frontend 修改以及 workload fetcher 提 pr；@zxustc
2. metric collector 的实现和提 pr； @zwzhang0107
3. 优化 checkpoint 接口，生成 controller；@xueche

**Comment by zxustc** [comment]
---todo list---
快速迭代实现 metric collector 落地内部场景；@zxustc
frontend 修改以及 workload fetcher 提 pr；@zxustc
metric collector 的实现和提 pr； @zwzhang0107
优化 checkpoint 接口，生成 controller；@xueche
gpu 拓扑相关实现调研，6.4 社区周会；@xueche

**Comment by clay-wangzhi** [comment]
@zxustc  请问，现在资源画像和数据预测的能力已经支持了吗，有相关的文档介绍吗？

**Reply by ZiMengSheng** [reply]
ack-koordinator 当中有：https://help.aliyun.com/zh/ack/ack-managed-and-ack-dedicated/user-guide/resource-profiling?spm=a2c4g.11186623.0.0.6fff424eHTwh1u#task-2192631，目前还么有在开源中实现

**Reply by clay-wangzhi** [reply]
了解了，谢谢
